Aptoide/aptoide-client-v8
# [Task/APP-498] Fix dev (translations)

**What does this PR do?**

  Fix dev translations. The values folder had wrong files inside (non-xml)
  Fix String.

**Database changed?**

   No

**How should this be manually tested?**

  Check if String that were removed were not different or expected to being used somewhere else.
  Flow on how to test this or QA Tickets related to this use-case: [APP-498](https://aptoide.atlassian.net/browse/APP-498)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-498](https://aptoide.atlassian.net/browse/APP-498)

**Code Review Checklist**
